extensions/tokenizers/rust/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "djl"
+version = "0.1.0"
+authors = ["Frank Liu <frankfliu2000@gmail.com>"]
+edition = "2021"
+
+[dependencies]
+jni = "0.19.0"
+tokenizers = { path = "../tokenizers/tokenizers", version = "*" }
```

Comment:
I'd fix the version here, unless you want to run this in a CI and spot breaking changes.

Reply:
The version of tokenizers is defined in `gradle.perperties` file, we always clone a specific version of tokenizer to build the project.